src/app-logic/l10n.js

```diff
@@ -10,28 +10,38 @@ import {
   PSEUDO_STRATEGIES_DIRECTION,
 } from 'firefox-profiler/utils/l10n-pseudo';
 
-// This contains the locales we support in the production. Don't forget to update
-// the array RTL_LOCALES when adding a RTL locale, if necessary.
-// AVAILABLE_STAGING_LOCALES is replaced, using webpack's DefinePlugin, by all
-// available locales, when running with L10N=1, using the "l10n" versions of the
-// yarn scripts. Especially this is done when building and deploying the l10n
-// branch in netlify.
-export const AVAILABLE_LOCALES: Array<string> = AVAILABLE_STAGING_LOCALES || [
-  'de',
-  'el',
-  'en-GB',
-  'en-US',
-  'es-CL',
-  'fr',
-  'ia',
-  'it',
-  'nl',
-  'pt-BR',
-  'sv-SE',
-  'uk',
-  'zh-CN',
-  'zh-TW',
-];
+// This object contains the locales that are complete at at least 90% and
+// that enable by default. Don't forget to update the array RTL_LOCALES when
+// adding a RTL locale, if necessary.
+// The localized names are copied from the select box at the bottom of
+// https://addons.mozilla.org/. We use the language names in their own language.
+export const AVAILABLE_LOCALES_TO_LOCALIZED_NAMES = {
+  de: 'Deutsch',
+  el: 'Ελληνικά',
+  'en-GB': 'English (British)',
+  'en-US': 'English (US)',
+  'es-CL': 'Español', // Use "Español (de Chile)" once we have more spanish versions
+  fr: 'Français',
+  ia: 'Interlingua',
+  it: 'Italiano',
+  nl: 'Nederlands',
+  'pt-BR': 'Português', // Use "Português (do Brasil)" once we have more portuguese versions
```

Comment:
nit: I think it's better to explicitly call out the actual language, especially for Brazilian Portuguese. For Spanish, we can probably get away with it.